ldms/python/ldmsd/ldmsd_yaml_parser

```diff
@@ -42,7 +42,7 @@ if __name__ == "__main__":
         sys.exit(errno.EINVAL)
 
     if args.daemon_name:
-        ldmsd_cfg_str = cluster.daemon_config(args.ldms_config, args.daemon_name.rstrip('0'))
+        ldmsd_cfg_str = cluster.daemon_config(args.ldms_config, args.daemon_name.rstrip('\0'))
```

Comment:
This is not correct either. I would recommend dropping rstrip() altogether (this is string from the command line, so this isn't needed), or use just "rstrip()" or "strip()". @baallan has a better fix in #2122.